### DISCUSSION TITLE
you have maximum limit how far you can claim town from other town how to fix this?

### DISCUSSION THREAD

**Discussion by SecretZudan** [TownyAdvanced/Towny#5663] [Q&A] [answered]
which line i need to modify to fix this?

**Comment by LlmDl** [comment, marked as answer]
The config contains min_distance settings, which is what I think you're asking about.

**Reply by SecretZudan** [reply]
which one?

**Reply by SecretZudan** [reply]
my friend wants(aleqo1223 ) wants to create town where he is but can't http://188.129.172.88:27891/

**Reply by SecretZudan** [reply]
for you to know better :)

**Reply by LlmDl** [reply]
All of the min_distance settings have an effect on different things. Read the config's comments.

**Reply by SecretZudan** [reply]
i did and idk what homeblock is :/

**Reply by SecretZudan** [reply]
and maybe its max_distance?

**Reply by LlmDl** [reply]
If they are trying to claim far away it would be max_distance.

**Reply by SecretZudan** [reply]
yep its max_distance and i think it was homeblock problem (i think)
